broker/src/main/java/io/zeebe/broker/system/monitoring/DiskSpaceUsageMonitor.java

```diff
@@ -34,6 +34,7 @@ public DiskSpaceUsageMonitor(final DataCfg dataCfg) {
 
   @Override
   protected void onActorStarted() {
+    checkDiskUsageAndNotifyListeners();
```

Comment:
This comment is actually for line 30 `this.minFreeDiskSpaceRequired = dataCfg.getFreeDiskSpaceCommandWatermark();`

I am pretty (albeit not a 100%) sure that this can be wrong. During out game days one of the interventions that the cloud team was able to do was to increase the PVC of the running broker. That means in the cloud the disk can grow (or shrink) at runtime, so this should become a supplier instead of a cached vale.

Reply:
Addendum: alistair is 99% sure this is done at runtime

Reply:
Oh is it? I thought we have to restart the pod for the change to be in effect.

Reply:
Here what the cloud guys had to say: https://camunda.slack.com/archives/CKZK2E7RP/p1599037325041500

Reply:
I will create a new issue for it because it is unrelated to this PR.

Reply:
https://github.com/zeebe-io/zeebe/issues/5280